communicating_vessels.py

```diff
@@ -0,0 +1,16 @@
+def interleave(*organs):
+    # Create an empty list to store the interleaved elements
```

Comment:
I would use documentation instead, comments make it look messy
https://swimm.io/blog/code-comments-vs-documentation/

Reply:
I fixed it